task/buildah/0.1/README.md

```diff
@@ -0,0 +1,28 @@
+# buildah task
+
+## Description:
+Buildah Task builds source into a container image using Project Atomic's Buildah build tool.It uses Buildah's support for building from Dockerfiles, using its buildah bud command.This command executes the directives in the Dockerfile to assemble a container image, then pushes that image to a container registry.
+
+## Params:
+
+| name                            | description |
+|---------------------------------| --- |
+|SKIP_CHECKS | Skip checks against built image|
+|IMAGE|Reference of the image buildah will produce.|
+|BUILDER_IMAGE|The location of the buildah builder image.|
+|DOCKERFILE|Path to the Dockerfile to build.|
+|CONTEXT|Path to the directory to use as context.|
+|TLSVERIFY|Verify the TLS on the registry endpoint (for push/pull to a non-TLS registry)|
```

Comment:
same as above

Reply:
removed this task in prior update so resolving comments re: buildah